.github/workflows/publish-crates.yml

```
@@ -6,11 +6,13 @@ jobs:
     runs-on: ubuntu-latest
     environment: crate-publish
     permissions:
```

actionlint (1.7.8)

string should not be empty

(syntax-check)